assets/js/buttons-examples.js

```diff
@@ -516,6 +516,16 @@ $(document).ready(function () {
             },
           ],
         },
+        {
```

Comment:
Place this in the correct alphabetical order under Special

Reply:
done in my recent commit